src/index.css

```diff
@@ -2,4 +2,5 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
-}
+  background-color: #fafafa;
```

Comment:
you can create a inject global

Reply:
Ok. I missed the option of use injectGlobal, but with your talk yesteday now I know how to solve this and use styled components instead of this css.

Very nice talk, thank you 😃 .